src/frontend/src/lib/utils/token-group.utils.ts

```diff
@@ -81,7 +82,11 @@ export const groupTokensByTwin = (tokens: TokenUi[]): TokenUiOrGroupUi[] => {
 		return token;
 	});
 
-	return mappedTokensWithGroups.filter(
-		(t) => isTokenUiGroup(t) || !groupedTokenTwins.has(t.symbol)
-	);
+	return mappedTokensWithGroups
+		.filter((t) => isTokenUiGroup(t) || !groupedTokenTwins.has(t.symbol))
+		.sort(
```

Comment:
Can we have a test?

Reply:
Sorry, forgot! Just included, have a look please